src/phoenix/server/api/types/EvaluationSummary.py

```diff
@@ -20,16 +22,16 @@ class LabelFraction:
 class EvaluationSummary:
     count: int
     labels: Tuple[str, ...]
-    evaluations: Private[Tuple[pb.Evaluation, ...]]
+    annotations: Private[Sequence[annotation_type]]
```

Comment:
I think we should keep the domain language to `evaluation`. As I understand it, we use "annotation" to refer to LLM- or human-applied labels, but "evaluation" refers specifically to LLM-applied labels, which is what we have in this case. Does that track with your understanding @mikeldking @RogerHYang?

Reply:
Either way seems fine to me. We can also think of "annotation" as the step after the act of "evaluation", then in that case annotation is what we have here.